src/super/i-static-page/modules/theme/factory.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * V4Fire Client Core
+ * https://github.com/V4Fire/Client
+ *
+ * Released under the MIT license
+ * https://github.com/V4Fire/Client/blob/master/LICENSE
+ */
+
+import ThemeManager from 'super/i-static-page/modules/theme/theme-manager';
+
+/**
+ * Returns class instance to manage interface themes
```

Comment:
Returns a class instance to manage interface themes if that functionality is available